scripts/test/MultiPlotting/MultiPlottingContext_test.py

```diff
@@ -111,7 +111,7 @@ def test_that_remove_line_calls_the_correct_functions(self):
         self.context.remove_line("plot", "line name")
 
         self.assertEqual(1, self.context.subplots["plot"].removeLine.call_count)
-        self.assertEqual(1, self.context.subplots["plot"].redraw_annotations.call_count)
+        self.assertEqual(0, self.context.subplots["plot"].redraw_annotations.call_count)
```

Comment:
why has this changed to 0?

Reply:
Because in commit `f85b754` the redraw annotations was removed and this updates the test as I forgot to do it together.